ddp-workspace/projects/ddp-dsm-ui/src/app/services/auth.service.ts

```diff
@@ -129,9 +130,10 @@ export class Auth {
       next: response => dsmResponse = response,
       complete: () => {
         const dsmToken = dsmResponse.dsmToken;
-        localStorage.setItem(SessionService.DSM_TOKEN_NAME, dsmToken);
```

Comment:
Moved this line to `session.service.ts` (https://github.com/broadinstitute/ddp-angular/pull/1102/files#:~:text=localStorage.setItem(SessionService.DSM_TOKEN_NAME%2C%20token)%3B)